py3status/modules/nvidia_temp.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+"""
+Module for displaying NVIDIA GPU temperature
+
```

Comment:
Here we miss a documentation about configuration parameters please (see the example module)